src/main/java/uk/gov/hmcts/sptribs/systemupdate/convert/CaseDetailsListConverter.java

```diff
@@ -14,8 +14,12 @@
 @Slf4j
 public class CaseDetailsListConverter {
 
+    private final CaseDetailsConverter caseDetailsConverter;
+
     @Autowired
-    private CaseDetailsConverter caseDetailsConverter;
```

Comment:
A new test case can be added to CaseDetailsListConverterTest to test the exception scenario.